docs/build/tutorials/platform/adjusting-gas-price-during-high-network-activity.md

```diff
@@ -0,0 +1,101 @@
+# Adjusting gas price during high network activity
```

Comment:
Please add a description field (see [this doc](https://github.com/ava-labs/avalanche-docs/blob/master/docs/build/tutorials/platform/launch-your-ethereum-dapp.md) as an examples) for SEO, otherwise, the 2nd paragraph will be taken as the description.